client/signup/main.jsx

```diff
@@ -45,7 +45,8 @@ import * as oauthToken from 'lib/oauth-token';
 /**
  * Constants
  */
-const MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED = 8000;
+const MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED_FREE = 8000;
+const MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED_PREMIUM = 3000;
```

Comment:
IMHO we shouldn't worry the user about email confirmation at all at this step. I saw that Delta had proposed an in-context notice about email confirmation in the calypso editor right by the publish button, where it's most relevant (@meremagee). Once we drop the email confirmation text (in a future PR) we should drop this wait time as well.